codespell.py

```diff
@@ -80,14 +81,108 @@ def load_text_file(filename):
     return output
 
 
+def checkWords(spell_checker, words):
+    """Check each word and report False if at least one has error."""
+    for word in words:
+        if not spell_checker.check(word):
+            return False
+    return True
+
+
+def spell_check_comment(
+    spell_checker: SpellChecker,
+    c: comment_parser.common.Comment,
+    prefixes: list[str] = [],
+    output_lvl=2,
+) -> list[str]:
+    """Check comment and return list of identified issues if any."""
+
+    if output_lvl > 1:
+        print(f"Line {c.line_number()}: {c}")
+
+    mistakes = []
+    spell_checker.set_text(c.text())
+
+    for error in spell_checker:
+        if output_lvl > 1:
+            print(f"Error: {error.word}")
+
+        error_word = error.word
+
+        valid = False
+
+        # Check for contractions
+        for contraction in ["'d", "'s", "'th"]:
```

Comment:
IMO, it is usually cleaner to have constants such as these defined at the beginning, so that we do not find them deep into the code, may be easier to locate or re-use. Maybe a preference.

Reply:
See the following pull request:
* https://github.com/SimpleITK/SimpleITKSpellChecking/pull/44